dcc-portal-ui/INTERNATIONALIZATION.md

```diff
@@ -0,0 +1,226 @@
+# ICGC DCC - Portal UI Internationalization
+
+## Overview
+
+For the Internationalization of ICGC DCC Portal we are using [angular-gettext](https://github.com/rubenv/angular-gettext). This tool extracts the translatable strings from the application
+and adds them into a standard `.pot` template file. [Development](#development) section explains more detail into how to setup the tool into work enviornment. once the `.pot` has been
+generated, the translator can use it as a resource and translate the strings. Please look at the [Translation](#translation) section for more details. The translation should crate a new
+`.po` file which would then be used to compile strings and translations into a JavaScript file. 
+
+## Development
+
+These are the instruction on how to setup angular-gettext, integrate into the application. For the English version Portal is using already generated `strings.en_CA.po` 
+and compiled translation `app/scripts/translations/js/translations.js` file. Any string changes should be done on `.po` file. Please look at [Integration](#integration) section for more
+details.
+
+### Setup
+
+- Initial setup for angular-gettext has been done for Portal. You can just run `bower install` command from the root directory of **dcc-portal-ui** and it will install needed module to start developing. Below information is just for knowledge. 
+
+- Install the angular-gettext plugin
```

Comment:
Hmm, our cost/benefit functions seem to greatly differ

We're essentially duplicating their documentation, I think we should at most link to it instead 

"To see how internationalization was setup, see angular-gettext's [Get Started page](https://angular-gettext.rocketeer.be/dev-guide/install/)". Although even that seems unnecessary since we're already linking to angular-gettext in the first sentence

We also don't have instructions on how to setup Angular, Karma, d3.